### PR TITLE
Deploy to alfajores

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -21,14 +21,17 @@ jobs:
       - run: yarn
       - run: yarn build
       - run: yarn test
-  lint:
```
Comment (gnardini) [inline .github/workflows/workflow.yaml:24]: Just curious, why did you delete the lint job?

**Review by gnardini — APPROVED** [review]
Left a Q regarding linting :)

File: .github/workflows/workflow.yaml (hunk 1 above)
Comment (jh2oman) [reply .github/workflows/workflow.yaml:24]: Looks like `yarn test` also runs eslint https://github.com/valora-inc/blockchain-api/blob/main/package.json#L9

**Review by jh2oman — APPROVED** [review]

File: .github/workflows/workflow.yaml (hunk 1 above)
Comment (silasbw) [reply .github/workflows/workflow.yaml:24]: I should have pulled that into a separate PR.  `yarn test` includes linting (and the `actions/setup-node@v2` action automatically parses the errors and annotates the PR).
